### ISSUE TITLE
Guidance on version control with shared OneDrive files

### ISSUE THREAD

**Issue by sashahafner** [AU-BCE-EE/guidance#3]
OneDrive can be handy but if we keep multiple copies of a shared document the convenient shortcuts that show up when opening MS Word can send collaborators to an old file.
Then you have a version control nightmare.
Can we come up with a good general approach and document it here?

**Comment by JohannaPedersen** [comment]
I am not the fastest one at replying to issues I can see, but better late than never. 

For the last manuscript I was lead on I did as you suggest below, it worked without problems!  

> How about this?
> 
> * Keep 1 main version of the shared document in a fixed location and fixed name
> * If you want to create separate checkpoint or backup versions after some event or major change etc. make a copy of the main file and paste/move it to a sub-directory. Considering changing its name (e.g., append a date) for clarity.
> 
> This way users will only ever have a Word shortcut to a single version, which will always be the latest one.

**Comment by fdalby** [comment]
> I am not the fastest one at replying to issues I can see, but better late than never.

Oops, I have only just seen this as well. Sorry!

**Comment by sashahafner** [comment]
> For the last manuscript I was lead on I did as you suggest below, it worked without problems!

Great.

Here is what that looks like for me, maybe just because it is easier to see than read:

![image](https://github.com/AU-BCE-EE/guidance/assets/35272876/d9cd4371-36d9-441a-964c-c22138b277b0)

And then in backup subdir:

![image](https://github.com/AU-BCE-EE/guidance/assets/35272876/081d6a22-7210-4957-be3f-2a8c529678c7)

**Comment by sashahafner** [comment]
But I have realized "manuscript" isn't a good name. 
Fine if we/collaborators always browse to it through Windows Explorer or some other file manager.
But it isn't handy to have a bunch of shortcuts to "manuscript.docx" in MS Word.
So how about being sure to use a descriptive name as well, like this?

![image](https://github.com/AU-BCE-EE/guidance/assets/35272876/2c0ca0be-a07f-4053-8446-13622013bb84)

Again, this is only about shared files.

**Comment by JohannaPedersen** [comment]
I agree it is good to have a short descriptive name as it can also cause issues with office if you want to have several files open with the same name.

**Comment by sashahafner** [comment]
Now we have this:

https://github.com/AU-BCE-EE/guidance/blob/main/OneDrive.md

It is a proposal.
Could be edited or we could have multiple versions, and could refer students etc. to the approach the project leader prefers.
